dao/chat.go

```diff
@@ -104,7 +103,7 @@ func (d chatDao) GetVisibleChats(userID uint, streamID uint) ([]model.Chat, erro
 // Number of likes are inserted and the user's like status is determined
 func (d chatDao) GetAllChats(userID uint, streamID uint) ([]model.Chat, error) {
 	var chats []model.Chat
-	query := DB.Preload("Replies").Preload("UserLikes").Preload("AddressedToUsers").Find(&chats, "stream_id = ?", streamID)
+	query := DB.Preload("Replies").Preload("Reactions").Preload("AddressedToUsers").Find(&chats, "stream_id = ?", streamID)
```

Comment:
Out of curiosity, have you done some performance testing with lots of chats (in the hundreds for one stream)?

Don't worry though, this should be fine.

Reply:
No benchmarking done yet, would be interesting tho! :)